src/github.com/matrix-org/dendrite/syncserver/storage/output_room_events_table.go

```diff
@@ -232,7 +248,14 @@ func (s *outputRoomEventsStatements) Events(txn *sql.Tx, eventIDs []string) ([]g
 	return result, nil
 }
 
-func rowsToEvents(rows *sql.Rows) ([]gomatrixserverlib.Event, error) {
+type rowOrdering uint8
+
+const (
+	rowOrderForward = rowOrdering(iota)
+	rowOrderReverse
+)
```

Comment:
I don't understand what you mean. I want the latest X messages, ordered from old -> new. I can't do that with a single `ORDER BY`, as getting the latest X messages is done by ordering the `id` from high -> low, but then within those returned results I want to reverse them so it's low -> high.